pkg/controller/direct/secretmanager/mapper.generated.go

```diff
@@ -0,0 +1,203 @@
+// Copyright 2024 Google LLC
```

Comment:
Nit: we should add these to `.gitattributes` to indicate they are generated code (and probably add a "this is generated code" comment)!

Reply:
Thank you for updating the .gitattributes!